rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/activity/CovidPatientProfileActivityTest.java

```diff
@@ -0,0 +1,45 @@
+package io.ona.rdt.robolectric.activity;
+
+import android.app.Activity;
+import android.content.Intent;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.robolectric.Robolectric;
+import org.robolectric.annotation.Config;
+
+import io.ona.rdt.activity.CovidPatientProfileActivity;
+import io.ona.rdt.domain.Patient;
+import io.ona.rdt.robolectric.shadow.MockCounter;
+import io.ona.rdt.robolectric.shadow.UtilsShadow;
+import io.ona.rdt.util.Constants;
+
+@Config(shadows = {UtilsShadow.class})
+public class CovidPatientProfileActivityTest extends ActivityRobolectricTest {
+
+    private CovidPatientProfileActivity covidPatientProfileActivity;
+
+    @Before
+    public void setUp() {
+
+        MockCounter counter = new MockCounter();
+        UtilsShadow.setMockCounter(counter);
+        Assert.assertEquals(0, UtilsShadow.getMockCounter().getCount());
```

Comment:
This should move here https://github.com/onaio/rdt-standard/pull/945/files#diff-6a71c852013ca983178a0f9a4a06281c6f45ad4be0c2ff8feec8d3d60a64c968R38.

Also null out the mockcounter after the test is done.

Reply:
this will not work if we move the assertion here because `Utils.verifyUserAuthorization` is calling in `onResume` method.

Reply:
you can call onResume in the test.

Reply:
fixed.